examples/data_preprocessing/mimic_dataset/prepare_mimic_cxr.ipynb

```diff
@@ -0,0 +1,1862 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# merge labels with records and reports\n",
+    "input_list_labels_pd = pd.merge(input_list_pd, \n",
+    "                                    labels_chexpert.iloc[:,[0,1,16]], \n",
+    "                                    how = 'left', \n",
+    "                                    on = ['study_id','subject_id'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 68,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "classes proportions: label\n",
+      "Atelectasis                   0.077801\n",
+      "Cardiomegaly                  0.070539\n",
+      "Consolidation                 0.016598\n",
+      "Edema                         0.048755\n",
+      "Enlarged Cardiomediastinum    0.009336\n",
+      "Fracture                      0.012448\n",
+      "Lung Lesion                   0.007261\n",
+      "Lung Opacity                  0.103734\n",
+      "No Finding                    0.401452\n",
+      "Pleural Effusion              0.117220\n",
+      "Pleural Other                 0.004149\n",
+      "Pneumonia                     0.031120\n",
+      "Pneumothorax                  0.013485\n",
+      "Support Devices               0.086100\n",
+      "dtype: float64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"classes proportions:\", \n",
+    "      input_list_labels_pd.groupby('label').size()/len(input_list_labels_pd))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are 14 classes, where about 40% of the studies are assigned to the class 'No finding'. We should keep in mind that we are dealing with an unbalanced dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lastly, we change the class names to their indices and convert the pandas dataframe to a numpy array."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 69,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████| 964/964 [00:00<00:00, 5965.69it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Changing names to indices\n",
+    "for i in tqdm(range(len(input_list_labels_pd))):\n",
+    " input_list_labels_pd.iloc[i,5] = labels.get(input_list_labels_pd.iloc[i,5])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 70,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# convert to numpy\n",
+    "input_list_labels = input_list_labels_pd.to_numpy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Finetuning"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we set a number of images which should be used for finetuning. If the number of images downloaded is less than 750, all images are used, otherwise we will use 750."
```

Comment:
```suggestion
    "First, we set a number of images, which should be used for finetuning. If the number of images downloaded is less than 750, all images are used, otherwise we will use 750."
```